assets/texts/sk/Changelog/1.2.1.html

```diff
@@ -0,0 +1,5 @@
+<ul>
```

Comment:
If you have untranslated files, then please just remove them. (i.e. no English files in the `sk` dir) AMO automatically falls back to English. (and I need to upload it manually, anyway)